main.go

```diff
@@ -1,30 +1,75 @@
 package main
 
 import (
+	"runtime"
+
+	"github.com/gin-contrib/gzip"
 	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
 
+	messageConfig "arkavidia-backend-8.0/competition/config/message"
 	"arkavidia-backend-8.0/competition/middlewares"
 	"arkavidia-backend-8.0/competition/routes"
-	databaseService "arkavidia-backend-8.0/competition/services/database"
-	storageService "arkavidia-backend-8.0/competition/services/storage"
+	"arkavidia-backend-8.0/competition/utils/mail"
+	"arkavidia-backend-8.0/competition/utils/validation"
 )
 
+// TODO: Gunakan gzip untuk mengkompresi size HTTP Response
+// REFERENCE: https://dasarpemrogramangolang.novalagung.com/C-http-gzip-compression.html
+// ASSIGNED TO: @rayhankinan
+// STATUS: DONE
+
+// TODO: Tambahkan secure middleware untuk menambah security
+// REFERENCE: https://dasarpemrogramangolang.novalagung.com/C-secure-middleware.html
+// REFERENCE: https://dasarpemrogramangolang.novalagung.com/C-https-tls.html
+// NOTES DARI GARE: Pake openssl aja buat generate certificate
```

Comment:
Its more secure to buy certificate fron CA but for this case lets just self signed one :) (use OpenSSL ajah)

Reply:
Ashiapp